### ISSUE TITLE
Why records' id are keep changing every time I restart the server?

### ISSUE THREAD

**Issue by yf-hk** [adityamukho/sails-nedb#1]
I found that when sails restarted, every time the id field of all records will change, which is a big problem because I have a few models which have associations. So they are using id field to maintain the relation and it will break those associations during restart. Is it something  related to nedb itself or we can fix it in sails-nedb adapter?
If it's not fixable, I'm considering to write an adapter for tingodb, it's much more mature than nedb and provides almost compatible apis as mongodb.

**Comment by adityamukho** [comment]
I don't think this is an issue with nedb itself. Will need to investigate further. Am I right in assuming, from your description, that the entities themselves are not lost and recreated upon restart?

**Comment by yf-hk** [comment]
@adityamukho Thanks for your reply. Yes, all other values keep the same, but id field is keep changing. By the way, I've finished sails-tingo adapter, welcome to test it :)
https://github.com/andyhu/sails-tingo

**Comment by adityamukho** [comment]
Thanks for the contribution. This benefits the sails community greatly!

I have observed the issue you have mentioned when running my adapter using a recent version of nedb, whereas an older version did not seem to have this issue. Still need to dig deeper to figure out if this is an nedb-specific issue or if my adapter is acting funny.

I am marking this as a bug for now.

**Comment by yf-hk** [comment]
Actually I have fixed those issues now. I rewrote most of the code. Here is the link
https://github.com/andyhu/waterline-nedb
Probably we could merge the two version in the future with different versions
